hedera-node/hedera-app-spi/src/main/java/module-info.java

```diff
@@ -5,6 +5,7 @@
     requires com.google.protobuf;
     requires com.swirlds.config;
     requires com.github.spotbugs.annotations;
+    requires com.hedera.hashgraph.protobuf.java.api;
```

Comment:
Why is this dependency needed? Seems odd, we should not rely on the old protobuf-classes anymore.

Reply:
oh, that looks like an error in the imports. Will have a look